heidrun/indiana_dc.rb

```diff
@@ -0,0 +1,64 @@
+Krikri::Mapper.define(:indiana_dc,
+                      :parser => Krikri::OaiDcParser)
+  provider :class => DPLA::MAP::Agent do
+    uri 'http://dp.la/api/contributor/indiana'
+    label 'Indiana Memory'
+    #I'm guessing here as to the naming of the above things
+  end
+
+  dataProvider :class => DPLA::MAP::Agent do
+    providedLabel record.field('dc:contributor')
+    end
+
+  isShownAt :class => DPLA::MAP::WebResource do
+    uri record.field('dc:identifier')
+  end
+
+  preview :class => DPLA::MAP::WebResource do
+    uri record.field('dc:source')
+    
+
+  originalRecord :class => DPLA::MAP::WebResource do
+    uri record_uri
+  end
+
+  sourceResource :class => DPLA::MAP::SourceResource do
+  
+  collection :class => DPLA::MAP::Collection, :each => record.field('xmlns:header', 'xmlns:setSpec'), :as => :coll do
+      title coll
+    end
+
+
+    creator :class => DPLA::MAP::Agent, :each => record.field('dc:creator'), :as => :creator do
+      providedLabel creator
+    end
+
+    date :class => DPLA::MAP::TimeSpan, :each => record.field('dc:date'), :as => :created do
+      providedLabel created
+    end
+
+    description record.field('dc:description')
+
+
+    dcformat record.field('dc:format')
+
```

Comment:
Do we need a mapping for `genre` here? From the spreadsheet:

> dc:type when the value matches 
> Book
> Film/Video
> Manuscript
> Maps
> Music
> Musical Score
> Newspapers
> Nonmusic
> Photograph/Pictorial Works
> Serial

This could also be something that could be dealt with in enrichment (mapping from any existing values for `dctype` to `genre`). I'm going to cc @no-reply because I think there's a philosophical angle he might want us to consider.